agent/config/runtime_test.go

```diff
@@ -4283,7 +4283,8 @@ func TestFullConfig(t *testing.T) {
 					"key": "sl3Dffu7",
 					"args": ["dltjDJ2a", "flEa7C2d"]
 				}
-			]
+			],
+			"raft_apply_timeout": "0s"
```

Comment:
In this test we are using some random values to make sure they come out properly. The problem with `0s` is that this is the default and we cannot tell if the conversion is working or not. This line and the next one in the hcl config should have the same random duration. eg. `20m44s`

Reply:
Ah I see. Thanks for pointing out! I fixed it in the latest branch. Could you take a look?